README.md

```diff
@@ -96,8 +96,8 @@ Example:
 
 ...
 
-const user = user.fakeExpect('name', 'age');
-const users = user.fakeExpect('name', 'age');
+const user = User.fakeExcept('name', 'age');
```

Comment:
@stevesweets This are instance level methods to let update `user` instance with changes.

Reply:
Ah, my mistake. Will review and update